cocotb/utils.py

```diff
@@ -118,7 +118,7 @@ def get_sim_steps(time, units=None):
         :exc:`ValueError`: If given *time* cannot be represented by simulator precision.
     """
     result = time
-    if units is not None:
+    if units not in (None, "step"):
```

Comment:
Is there any point supporting `None` here? It seems odd to introduce two ways to spell the same thing, unless our plan is to deprecate one of them.

I'd be tempted to change the default to `units="step"`.

Reply:
This change will need to be reflected in the docstring and newsfrag, as a reminder.

Reply:
Changed the default and deprecated None, with tests. Docstrings and newsfragments should be fine now too.